astronomer/airflow/version_check/plugin.py

```diff
@@ -32,6 +33,11 @@ def on_load(cls, *args, **kwargs):
         # Hook in to various places in Airflow in a slightly horrible way -- by
         # using functools.wraps and replacing the function.
 
+        if conf.getint("astronomer", "update_check_interval", fallback=24 * 60 * 60) == "0":
+            log.debug(
+                "Skipping running update_check_plugin as [astronomer] update_check_interval = 0")
```

Comment:
We should also not register the blueprint in the UI if this config is 0

Reply:
Will do

Reply:
Done @ashb